src/styles/blocks/stopwatch.scss

```
@@ -0,0 +1,70 @@
+.stopwatch {
+  position: absolute;
+  top: $half-percents;
+  left: $half-percents;
+  transform: translate(-50%, -50%);
+  width: $clock-width;
+  height: $clock-height;
+  border: 15px solid $frame-color;
+  border-radius: $half-percents;
+  background: url(~/images/stopwatch.png);
+  background-size: cover;
+
+  &:before {
+    content: "";
+    position: absolute;
+    top: $hand-top;
+    left: $half-percents;
+    width: 49%;
+    height: 2px;
+    background: $hand-sec-color;
+    animation: animate $sec-hand-time linear infinite;
+    transform-origin: left;
+  }
+
+  &:after {
+    content: "";
+    position: absolute;
+    top: $hand-top;
+    left: $half-percents;
+    width: 45%;
+    height: 3px;
+    background: $hand-min-color;
+    animation: animate $min-hand-time linear infinite;
+    transform-origin: left;
+  }
+
+  @keyframes animate {
+    0% {
+      transform: rotate(-90deg);
+    }
+    100% {
+      transform: rotate(270deg);
+    }
+  }
```

```suggestion
  @keyframes rotate {
    0% {
      transform: rotate(-90deg);
    }
    100% {
      transform: rotate(270deg);
    }
  }
```
Move animation to the separate file